editor/scene_tree_editor.cpp

```diff
@@ -483,6 +483,15 @@ void SceneTreeEditor::_test_update_tree() {
 	tree_dirty = true;
 }
 
+void SceneTreeEditor::_attempt_configuration_update() {
+	if (tree->is_editing()) {
+		update_timer->start();
+	}
+	else{
```

Comment:
Style issue, it should be `} else {` all on the same line.

Reply:
Fixed